Gemfile

```diff
@@ -26,6 +26,9 @@ group :test do
   gem 'rack', '~> 1.4.7'
   gem 'vcr'
   gem 'timecop'
+  gem 'pronto'
+  gem 'pronto-rubocop'
+  gem 'pronto-flay'
```

Comment:
Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `pronto-flay` should appear before `pronto-rubocop`.